onnxscript/version_converter/version_converter.py

```diff
@@ -1,55 +1,427 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT License.
+"""Convert the model to the specified ONNX opset version."""
+
 from __future__ import annotations
 
+import dataclasses
+import logging
+from typing import Callable, Sequence, Union
+
+import onnxscript.ir._convenience as _convenience
+import onnxscript.rewriter.pattern as orp
 from onnxscript import ir
-from onnxscript.version_converter._adapter_lib import pick_adapter_set
+
+logger = logging.getLogger(__name__)
+
+
+_ADAPTERS_18_19 = {
+    "Equal",
+    "AveragePool",
+    "Cast",
+    "CastLike",
+    "Constant",
+    "DequantizeLinear",
+    "Identity",
+    "If",
+    "Loop",
+    "Pad",
+    "QuantizeLinear",
+    "Reshape",
+    "Scan",
+    "Shape",
+    "Size",
+}
+
+
+_ADAPTERS_19_20 = {
+    "DFT",
```

Comment:
Ideally, we should not have to add this here and have the separate `@register` down below, just one will do. I think you should be able to drop these here for anything registered down below. (See other related comments below.)